molecule/default/tests/test_default.py

```diff
@@ -12,7 +13,69 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("x", [True])
-def test_packages(host, x):
-    """Run a dummy test, just to show what one would look like."""
-    assert x
+def test_packages(host):
+    """Verify that the expected packages are installed/uninstalled."""
+    assert host.package(
+        "systemd-resolved"
+    ).is_installed, "The package systemd-resolved is not installed."
+    assert not host.package(
+        "resolvconf"
+    ).is_installed, "The package resolvconf is installed."
+
+
+def test_symlink(host):
+    """Verify that /etc/resolv.conf is the expected symlink."""
+    f = host.file("/etc/resolv.conf")
+    assert f.is_symlink, "/etc/resolv.conf is not a symlink."
+
+    if host.system_info.distribution in ["amzn"]:
+        # /run/systemd/resolve/stub-resolv.conf is a symlink to
+        # /run/systemd/resolve/resolv.conf in AL2023, so the
+        # /etc/resolv.conf symlink resolves to the former.
```

Comment:
Maybe it's just me but when I read former here it seems like it would be referring to `/run/systemd/resolve/stub-resolv.conf`.

```suggestion
        # /etc/resolv.conf symlink resolves to the latter.
```